components/brave_shields/browser/base_brave_shields_service.cc

```diff
@@ -49,12 +49,6 @@ bool BaseBraveShieldsService::Start() {
   return false;
 }
 
-void BaseBraveShieldsService::Stop() {
```

Comment:
not used outside `ad_block_regional_service_manager.cc` where actually not needed